dbms/CMakeLists.txt

```diff
@@ -123,6 +123,10 @@ add_library(tiflash_common_io ${SPLIT_SHARED} ${tiflash_common_io_headers} ${tif
 target_link_libraries(tiflash_common_io PUBLIC process_metrics)
 target_include_directories(tiflash_common_io BEFORE PUBLIC ${XXHASH_INCLUDE_DIR} ${TiFlash_SOURCE_DIR}/libs/libprocess_metrics/include)
 
+if (USE_QPL)
+    target_compile_definitions (tiflash_common_io PUBLIC USE_QPL)
+endif ()
+
```

Comment:
```suggestion
```

Reply:
We tried to remove these 3 lines, but USE_QPL cannot be passed to the compiler. Conditional compilation of USE_QPL will not work. So we restored them.

Reply:
I missed this comment. Withdrawing my lgtm. Let me take a look deeper.